template/src/App.vue

```diff
@@ -1,35 +1,48 @@
 <template>
   <div id="app">
     <img class="logo" src="./assets/logo.png">
+    {{#if router}}
+    <router-view></router-view>
```

Comment:
Hmm... this is a tough situation. On the one hand, if we're going to include Vue Router, I'd like to also set up some scalable conventions and demonstrate more router features, like including navigation with `<router-link>` above. As it is, `<hello>` is also a bit silly as a page-level component. I'd prefer a `<home>` component and one other component - maybe `<about>`. I also typically recommend that people only use the `components` folder for _reusable_ components, rather than page-level components. I think it's better to have `pages` and `layouts` directories, where wrapper components from `layouts` are used as the root for page-level components in `pages`. 

Adding all this would mean a great deal more complexity for this template however. @zigomir @LinusBorg What are your thoughts? I'm worried that with this proposal, the people who would really benefit from out-of-the-box Vue Router integration would still be left with most integration questions unanswered, as they could probably get this far just from copying and pasting from the Getting Starting guide in the router docs. But adding a more complete Vue Router integration would be a maintenance nightmare.

Maybe we should just have a separate template that includes Vue + Vue Router + Vuex?